osu.Game.Rulesets.Osu/Difficulty/Preprocessing/OsuDifficultyHitObject.cs

```diff
@@ -121,20 +121,20 @@ private void setDistances(double clockRate)
                 //
                 // We'll try to better approximate the real movements a player will take in patterns following on from sliders. Consider the following slider-to-object patterns:
                 //
-                // 1.  <======o==>
+                // 1.  <======x==>
                 //            | /
-                //            o
+                //            x
```

Comment:
I still don't quite understand what this diagram is trying to explain, but as long as others can maybe it's okay.

Reply:
The diagrams show how the natural movement path of a player changes depending on what the angle of a follow-up circle is, and how using the lazy cursor end is completely fine in diagram 1 ("players will cut the slider short in order to move to the next object"), but will overestimate the perceived jump distance in diagram 2 ("players will follow the slider through to its visual extent").

I don't know how to better explain it.

Reply:
Am I supposed to know that there's a second object below the slider in the first diagram? So the "x" represents the position of the slider ball and also the position of the next object here?

Reply:
No, it represents the final position your cursor needs to be in a slider given the most lazy movements (following at the edge of the follow circle at all times) + the -36ms offset for the last slider tick.

Reply:
But the downwards movement implies there's something beneath it right? If so it's be nice to show that next object to give more context.

Reply:
I misunderstood what you meant, I thought you were referring to the "x" on the slider track.

How should I show that the "x" below the slider is another hitobject? Any suggestions? Would this be better?
![image](https://user-images.githubusercontent.com/1329837/145159865-95e8e124-55e5-43b3-9206-6deaa0524287.png)
I can't easily add the same text to the second diagram...

Reply:
Maybe they should be shown using `o` after all, and the documentation should not mention anything about the cursor position. The cursor movement is shown by the single-width line, which should be ample to explain the concept here.

Might help to split the diagram out into two versions beside each other: one with the "lazy" movement and one with the "correct" movement (ie the one denoted by `/`). Feels weird showing both on the same diagram.

Reply:
I've restructured it, see if it's better now?

Reply:
Yep, much better.